edge_computing/image-base-install/ibi-edge-image-based-install.adoc

```diff
@@ -0,0 +1,32 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ibi-image-based-install_{context}"]
+= Performing a deployment for {sno} clusters using zero-touch-configuration
+include::_attributes/common-attributes.adoc[]
+:context: ibi-edge-image-based-install
+
+toc::[]
+
+When a host preinstalled with {sno} arrives at a remote site, a technician can easily deploy the host in a matter of minutes using zero-touch-configuration (ZTC). ZTC uses the Image Based Install (IBI) Operator to configure the host and complete the deployment. 
+
+You must create configuration resources on the hub cluster containing the site-specific cluster configuration details for the host. The IBI Operator creates a configuration ISO from these resources. The Operator then boots the host with the configuration ISO attached. The host mounts the configuration ISO and runs the reconfiguration process. When the reconfiguration completes, the {sno} cluster is ready.
+
+[NOTE]
+====
+You must create a distinct configuration resources for each host. 
+====
+
+The following is a high-level overview for performing a deployment for {sno} clusters using zero-touch-configuration:
+
+* Install the IBI Operator on the hub cluster.
+* Create the following site-specific configurations configurations in the hub cluster:
+** `BareMetalHost`: Describes the physical host and its properties, such as the provisioning and hardware configuration.
+** `ImageClusterInstall`: References the the bare-metal host, deployment and image set resources.
+** `ClusterImageSet`: Describes the release images to use for the cluster.
+** `ClusterDeployment`: Describes networking, authentication, and platform-specific settings.
+** `ManagedCluster`: Describes cluster details to enable {rh-rhacm-first} to register and manage the host.
+// Config map?
```

Comment:
You'll need a config map for specific requirements (static networking, for example), but for the most basic case it wouldn't be needed.

Reply:
I added ConfigMap as an optional resource. Thanks.